packages/loader/container-loader/src/container.ts

```diff
@@ -1425,6 +1387,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             case MessageType.Operation:
             case MessageType.RemoteHelp:
             case MessageType.Summarize:
+            case MessageType.BlobAttach:
```

Comment:
Sorry if there was confusion on my part, but SPO wants to have blob info as metadata on op, not as a new type.
I'm sure I've left a comment on that in old PR with observation that we do not have good way of providing metadata through layers. As an example, I pointed to "prepareSend" handler in DataStoreRuntime that is used to augment ops with metadata. I believe we need to revisit it and have natural path from container runtime to provide metadata on ops.

This can be done as separate PR, please get the working (smaller) pieces in as they become finished!